pkgs/development/python-modules/numpy-groupies/default.nix

```diff
@@ -38,8 +37,6 @@ buildPythonPackage rec {
 
   pythonImportsCheck = [ "numpy_groupies" ];
 
-  passthru.updateScript = gitUpdater { tagPrefix = "v"; };
```

Comment:
Sorry, this should have been `unstableGitUpdater`.  Though it should either say this is not necessary (this is from Git, not PyPI) somewhere in https://github.com/NixOS/nixpkgs/blob/0da03f4801250961bc42d177711efaa4b79cec3e/doc/languages-frameworks/python.section.md so that one doesn't read https://github.com/NixOS/nixpkgs/blob/0da03f4801250961bc42d177711efaa4b79cec3e/pkgs/README.md#automatic-package-updates and then think it should be added to all packages.